core/src/main/java/com/google/common/truth/PrimitiveByteArraySubject.java

```diff
@@ -69,6 +65,95 @@ public void isEqualTo(Object expected) {
     }
   }
 
+  private String getArrayDiffMessage(byte[] actual, byte[] expected) {
+
+    int MAX_ELEMENTS = 10;
```

Comment:
Hi @dinesh707, apologies, I missed this the first time round. Could you make this variable either "camelCase" or a `private static final` field, as mandated by the Google Java style guide, please. :)